infrastructure/terraform/components/reporting/null_resource_dates_view.tf

```diff
@@ -0,0 +1,13 @@
+resource "null_resource" "dates_view" {
+  triggers = {
+    sql = filesha256("${path.module}/scripts/sql/views/dates.sql")
+  }
+  provisioner "local-exec" {
+    command = <<EOT
+      ${path.module}/scripts/create_replace_view.sh \
```

Comment:
odd indenting, is this a github-ism?

Reply:
Might be a Pullan-ism, but is consistent with a number of similar files that were already approved.

Reply:
Suggest let this one through, and tidy them all up en-masse if necessary.